bodhi/server/models.py

```diff
@@ -2617,13 +2617,16 @@ def set_request(self, db, action, username):
         topic = u'update.request.%s' % action
         notifications.publish(topic=topic, msg=dict(update=self, agent=username))
 
-    def waive_test_results(self, username, comment=None):
+    def waive_test_results(self, username, comment=None, waive_tests=None):
```

Comment:
You could just name the parameter ```tests```, which would read more naturally since the method name already implies that we are waiving tests.

Reply:
Will do